src/components/Delete.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Button } from '@mui/material';
+import { deleteEntryById } from '../services/entries';
+
+export default function Delete(update) {
```

Comment:
I am unsure how to utilize the delete button, I was unable to click an entry or delete anything with this button.